### PR TITLE
Easier setup

### PR DESCRIPTION
**Pull request by fjsj**
Friendlier setup

## Description
- Updated `README`
- `.env.example` file
- `eslint` on pre-commit
- Better error message on `manage.py` if missing `.env`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by aericson** [comment]
FYI Tests didn't pass because the bootstrap-loader broke, #134 fixes it.

File: README.md [hunk 1]
```diff
@@ -97,7 +87,24 @@ Add high level dependecies to `requirements-to-freeze.txt` and `pip freeze > req
...
 
+## Contributing
+
+### How to test `django-admin startproject`
+
+If you made changes to this boilerplate and want to test them, commit your changes and use `git archive -o boilerplate.zip HEAD` to create the template zip. Then, do a `cd ..` and a `django-admin startproject theprojectname --extension py,yml,json --name Procfile,README.md,.env.example --template=django-react-boilerplate/boilerplate.zip` to test the project bootstrap.
+
+### How to test Heroku deployment
+
+Push your changes to a branch and visit `https://dashboard.heroku.com/new?template=https://github.com/fill-org-or-user/fill-project-repo-name/tree/fill-branch` (replace all `fill`).
```
Comment (aericson) [inline README.md]: Maybe (replace all `fill-*`) makes it more visible.

File: README.md [hunk 2]
```diff
@@ -97,7 +87,24 @@ Add high level dependecies to `requirements-to-freeze.txt` and `pip freeze > req
...
+
+### How to add a 'Deploy to Heroku' button
+
+Read [this](https://devcenter.heroku.com/articles/heroku-button#adding-the-heroku-button).
+
+P.S. if you want to deploy in a different way please take a look the `app.json` file for what needs to be configured.
```
Comment (aericson) [inline README.md]: `please take a look at the` or `please look the`

**Comment by fjsj** [comment]
Thanks for the review @aericson ! Appreciate your help.

**Pull request merged**